.circleci/config.yml

```diff
@@ -447,6 +428,21 @@ commands:
     steps:
       - run: python .circleci/scripts/wait_for_master.py http://<<parameters.host>>:<<parameters.port>>
 
+  upload-cloud-watch-to-s3:
+    parameters:
+      bucket:
+        type: string
+        default: det-ci-cloud-watch-logs
```

Comment:
Non-blocking: I usually do the optional arguments after the required

Reply:
done